syft/pkg/cataloger/cataloger.go

```diff
@@ -46,6 +47,8 @@ func ImageCatalogers(cfg Config) []pkg.Cataloger {
 		binary.NewCataloger(),
 		deb.NewDpkgdbCataloger(),
 		dotnet.NewDotnetPortableExecutableCataloger(),
+		githubactions.NewActionUsageCataloger(),
+		githubactions.NewWorkflowUsageCataloger(),
```

Comment:
Are images a likely source for github actions?

Reply:
I figured there wasn't harm here, and though that's true, we try to only reference installed software for image catalogers, which this doesn't fit the bill. I'm going to back that out (keep it only to dir scans)